src/extensions/default/InlineColorEditor/ColorEditor.js

```diff
@@ -215,12 +234,12 @@ define(function (require, exports, module) {
     /** Add event listeners to the format button bar */
     ColorEditor.prototype._bindColorFormatToRadioButton = function (buttonClass, propertyName, value) {
         var handler,
-            _this = this;
+            self = this;
         handler = function (event) {
-            var colorObject, newColor, newFormat;
-            newFormat = $(event.currentTarget).html().toLowerCase().replace("%", "p");
-            newColor = _this.getColor();
-            colorObject = tinycolor(newColor);
+            var newFormat   = $(event.currentTarget).html().toLowerCase().replace("%", "p"),
+                newColor    = self.getColor().toString(),
```

Comment:
@redmunds That's a good question. As shown in this line you commented on, there are still uses of `toString()` in the extension itself as well as in the unit tests. I have run into some confusion in this PR over which to use, just to note that. As for unit tests, all I can do is test it using `getOriginalInput()`, but I'm not that it will fully work as the latter is not a drop-in replacement for `toString()`.